csv_dump.py

```diff
@@ -115,16 +117,25 @@ def get_tables_and_query_mapping():
     return tables_and_query
 
 if __name__ == '__main__':
-    db_name = 'postgres'
-    user = 'anju'
+
+    config = configparser.ConfigParser()
+    config.read('config.ini')
+    db_name = config['DatabaseSection']['database.dbname']
+    user = config['DatabaseSection']['database.user']
+
+    # check for args
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--active_timestamp", nargs='+', help="the timestamps where a VM was active", type=str)
+    args = parser.parse_args()
+    start_date = args.active_timestamp[0]
+    end_date = args.active_timestamp[1]
```

Comment:
I had been thinking two positional arguments after the toggles. Willing to discuss pros and cons of either.